pkg/utils/miniojob/minioJob.go

```diff
@@ -74,13 +82,58 @@ func File(fName string, ext string) FieldsFunc {
 				out.FileName = fName
 				out.FileExt = ext
 				out.FileContext = strings.TrimSpace(val)
+				out.ArgType = ArgTypeFile
 				return out, nil
 			}
 		}
 		return out, fmt.Errorf("file %s not found", fName)
 	}
 }
 
+// KeyValue - match key and putout the key, like endpoint="https://webhook-1.example.net"
+func KeyValue(key string) FieldsFunc {
+	return func(args map[string]string) (out Arg, err error) {
```

Comment:
Seems this function always returning an error

Reply:
> Seems this function always returning an error


No. 
```
                 val, ok := args[key]
		if !ok {
			return out, fmt.Errorf("key %s not found", key)
		}
		out.Command = fmt.Sprintf(`%s="%s"`, key, val)
```

Reply:
Line 104 `return out, fmt.Errorf("key %s not found", key)` ? Should this be `return out, nil`

Reply:
Yes